CAIPs/caip-27.md

```diff
@@ -12,20 +12,23 @@ requires: 2, 25, 171, 217
 
 ## Simple Summary
 
-CAIP-27 defines a JSON-RPC method for a wallet-connected application to invoke
-a wallet invoke an JSON-RPC method in a specified context defined by a valid
-[scopeObject][CAIP-217] and tagged with a [sessionId][CAIP-171] for maintaining session continuity.
+CAIP-27 defines a JSON-RPC method for a wallet-connected application to invoke a
+wallet invoke an JSON-RPC method in a specified context defined by a valid
+[scopeObject][CAIP-217] and, optionally, tagged with a [sessionId][CAIP-171] for
+maintaining session continuity when applicable.
 
 ## Abstract
 
 This proposal has the goal of defining a standard method for decentralized
-applications to invoke JSON-RPC methods from user agents (such as
-cryptocurrency wallets) directed to a given, previously-authorized target
-chain (such as nodes of a specific blockchain or consensus community within a
-protocol). It requires a valid [scopeObject][CAIP-217] and a valid
-[sessionId][CAIP-171] for interoperability and composability. These two
-properties MAY be inherited from a persistent session created by [CAIP-25][],
-but could also be used as part of other session management mechanisms.
+applications to invoke JSON-RPC methods from user agents (such as cryptocurrency
+wallets) directed to a given, previously-authorized target chain (such as nodes
+of a specific blockchain or consensus community within a protocol). It requires
+a valid [scopeObject][CAIP-217]. It MAY be tagged with a [sessionId][CAIP-171]
+for interoperability and composability if the [CAIP-25][] session which
+authorizes the request is keyed by a [sessionId][CAIP-171] (see [CAIP-316][] for
```

Comment:
nit. `if the caip-25 request which authorized the session...`

Reply:
https://github.com/ChainAgnostic/CAIPs/pull/320/commits/d16684712df6a12638a723cd92bca7716c391f23